src/views/Game/PlayerCards.tsx

```diff
@@ -541,12 +526,23 @@ function ScorePopup({ show, goban, color }: ScorePopupProps) {
     let first_points = 0;
     return (
         <div className="score_breakdown">
-            {color === "black" && !!goban.engine.handicap && (
-                <div>
-                    <span>{_("Handicap")}</span>
-                    <div>{goban.engine.handicap}</div>
+            {color === "black" && goban.engine.config.rules && (
+                <>
+                    <div className="summary">
+                        <span>
+                            {_("Rules")}: {rulesText(goban.engine.config.rules)}
+                            {rulesParens(goban.engine.config.rules)}
```

Comment:
I'm wondering if we really need the `{rulesParens()}` or if we can just delete this line and the `rulesParens` function.
- If we delete this line, the user can still discover that "NZ" means "Rules: New Zealand" by hovering over the "NZ".
- If we leave it in, it makes the pop-up unnecessarily busy with "Rule: New Zealand (NZ)". I've noticed this wraps if your window is small, but it didn't wrap before I added the parens.